java/vector/src/main/java/org/apache/arrow/vector/BaseValueVector.java

```diff
@@ -213,5 +213,13 @@ public static ArrowBuf transferBuffer(final ArrowBuf srcBuffer, final BufferAllo
     final ReferenceManager referenceManager = srcBuffer.getReferenceManager();
     return referenceManager.transferOwnership(srcBuffer, targetAllocator).getTransferredBuffer();
   }
+
+  @Override
+  public void copyFrom(int fromIndex, int thisIndex, ValueVector from) {
```

Comment:
should the default throw an Unsupported exception ?

Reply:
Thanks for the good suggestion. Throwing an exception is better.